azkaban-common/src/test/bash/test_util.shunit2

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env shunit2
```

Comment:
Thanks for contributing the first bash tests!
It will be great if the tests can be run automatically like java tests. 
But this is a big improvement already.